src/CommunityToolkit.Maui.Core/Interfaces/IAvatarView.shared.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>Avatar view interface.</summary>
+public interface IAvatarView : IBorderView, ILabel, Microsoft.Maui.IImage, IImageSource
+{
+	//// NOTE to implementer: Implement the following properties as public.
+
+	/// <summary>Gets a value indicating the avatar border colour.</summary>
+	Color BorderColor { get; }
+
+	/// <summary>Gets a value indicating the avatar border width.</summary>
+	double BorderWidth { get; }
+
+	/// <summary>Gets a value indicating the avatar corner radius (MAUI).</summary>
```

Comment:
Do we need MAUI in summary?

Reply:
The .NET MAUI type `CornerRadius` is not standard across all controls that implement a corner radius (they instead currently use the type `Int`).  From what I can see it is **only** available on the BoxView control.  I have updated the comment to include a cref.